requests_toolbelt/auth.py

```diff
@@ -64,3 +66,69 @@ def __call__(self, request):
 
         request.register_hook('response', self.handle_401)
         return request
+
+
+class HTTPProxyDigestAuth(HTTPDigestAuth):
+    """HTTP digest authentication between proxy"""
+    def __init__(self, *args, **kwargs):
+        """
+        Attributes:
+            stale_rejects: The number of rejects indicate that:
```

Comment:
First bit of feedback: Please make the doc-strings conform to the style (sphinx) of the rest of the library. We don't document `__init__` but this documentation about initialization parameters that are different from `requests.auth.HTTPDigestAuth` is appropriate for the class level docstring.